styles.css

```diff
@@ -421,10 +403,174 @@ body {
   padding-top: 10px;
 }
 
-.navigation ul {
+.navigation ul,
+#modalContainers {
   display: none;
 }
 
+.alert-Msg {
+  font-size: 12px;
+  font-weight: 600;
+  color: #fff;
+  padding-top: 12px;
+}
+
+/* Modal Design */
```

Comment:
-Excellent job with the backdrop blur 👨‍🎨 But let's fix left and right padding of the image like on the template.
- Please make sure you don't stretch the image, it is a very bad practice for the images. What you can do is, you can have a vertical overflow and user can go down to read the bottom section of the modal.

<img width="1407" alt="Screenshot 2023-02-03 at 11 21 53 PM" src="https://user-images.githubusercontent.com/25887253/216707451-67eabe02-64d4-452b-9a4f-31e00d2f9e58.png">
<img width="1407" alt="Screenshot 2023-02-03 at 11 22 28 PM" src="https://user-images.githubusercontent.com/25887253/216707459-7a751401-6ae9-4159-952a-bac69f9618ed.png">